cmd/lakefs/cmd/run.go

```diff
@@ -430,7 +430,8 @@ func checkRepos(ctx context.Context, logger logging.Logger, authMetadataManager
 		logger.Debug("lakeFS isn't initialized, skipping mismatched adapter checks")
 	} else {
 		logger.
-			WithField("adapter_type", blockStore.BlockstoreType()).
+			// TODO (gilo): uncomment this?
```

Comment:
Question: Why don't we just pass the responsibility to check the repo to the block adapter, provide it with the repository object and let it do this validation?